recipe/meta.yaml

```diff
@@ -21,6 +21,9 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    # Use pins to control cos6/cos7 match
+    - libgcc-ng  {{ libgcc }}
+    - libstdcxx-ng  {{ libstdcxx }}
```

Comment:
Why you need to pin them to match cos6/cos7?

Reply:
Anaconda just updated their compilers on x86. The new ones are now cos7 based instead of cos6 so there are glibc mismatches if you build on one and run on the other.
We already have pins on which compilers to use (for other reasons), and want to remain cos6 compatible for this release, so we are adding these libgcc and libstdcxx pins. It would be nice if Anaconda added a new `compiler` macro for cos6, but I'm not aware of it if they did.

Reply:
Understood. So it is for all projects, and it not strictly related to DALI, right?

Reply:
Correct.
And actually anything at all that builds against Anaconda's default channel needs to be careful since the glibc version in the toolchain's SYSROOT has been updated.